vanilla-framework/vanilla-framework
# Only publish contents of /scss to NPM

## Done

- Updated `.npmignore` file to ignore everything except the `/scss` folder + contents

## QA

- Pull code
- Run `npm pack`
- Check that the zip file only contains the `/scss` folder, `package.json`, `LICENSE` and `README.md`

## Details

Fixes #1836
